docs/sql/system.txt

```diff
@@ -371,34 +371,37 @@ thread_pools
-+-------------------------+----------------------------------------------+-------------+
-
-The cpu information values are cached for 1s. They might vary from the live terms. 
-Use the probe timestamps instead to get the terms which are measured at the time 
-of the collect. You could get the CPU usage for some period by analysing the start 
-and the end values using the "Probe-CPU" timestamp. Otherwise the usage could be 
-zero because of caching the measured values.
++---------------------------+----------------------------------------------+-------------+
+|       Column Name         |                 Description                  | Return Type |
++===========================+==============================================+=============+
+| ``os``                    | Operating system stats                       | ``Object``  |
++---------------------------+----------------------------------------------+-------------+
+| ``os['uptime']``          | System uptime in milliseconds                | ``Long``    |
++---------------------------+----------------------------------------------+-------------+
+| ``os['timestamp']``       | UNIX timestamp in millisecond resolution     | ``Long``    |
++---------------------------+----------------------------------------------+-------------+
+| ``os['cpu']``             | Information about CPU utilization            | ``Object``  |
++---------------------------+----------------------------------------------+-------------+
+| ``os['cpu']['system']``   | CPU time used by the system                  | ``Short``   |
++---------------------------+----------------------------------------------+-------------+
+| ``os['cpu']['user']``     | CPU time used by applications                | ``Short``   |
++---------------------------+----------------------------------------------+-------------+
+| ``os['cpu']['idle']``     | Idle CPU time                                | ``Short``   |
++---------------------------+----------------------------------------------+-------------+
+| ``os['cpu']['used']``     | Used CPU (system + user)                     | ``Short``   |
++---------------------------+----------------------------------------------+-------------+
+| ``os['cpu']['stolen']``   | The amount of CPU 'stolen' from this virtual | ``Short``   |
+|                           | machine by the hypervisor for other tasks.   |             |
++---------------------------+----------------------------------------------+-------------+
+| ``os['probe_timestamp']`` | The timestamp recorded at the collect of the | ``Long``    |
+|                           | CPU usage probe in milliseconds.             |             |
++---------------------------+----------------------------------------------+-------------+
+
+The cpu information values are cached for 1s. They might differ from the actual values. 
+Use the probe timestamp to get the time of the collect. You could get the CPU usage 
```

Comment:
The `probe_timestamp` is the system Unix timestamp at the moment of the probe collection.